src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetInputForm.vue

```diff
@@ -128,20 +128,22 @@ export default defineComponent<Props>({
         const widgetFormState = widgetFormStore.state;
 
         const state = reactive({
-            widgetConfig: computed(() => (props.widgetConfigId ? getWidgetConfig(props.widgetConfigId) : undefined)),
+            widgetConfig: computed<WidgetConfig|undefined>(() => (props.widgetConfigId ? getWidgetConfig(props.widgetConfigId) : undefined)),
             widgetOptionsJsonSchema: {} as JsonSchema,
-            //
             schemaFormData: {},
-            isSchemaFormValid: undefined,
-            isAllValid: computed(() => state.isSchemaFormValid && isTitleValid.value),
-            inheritableProperties: computed<string[]>(() => Object.entries(widgetFormState.inheritOptions ?? {})
+            // inherit
+            inheritOptions: {} as InheritOptions,
```

Comment:
nitpick: I think it should be deleted, if it's not used!

Reply:
thanks!